src/main/java/jnr/posix/LinuxPOSIX.java

```diff
@@ -22,16 +22,26 @@ final class LinuxPOSIX extends BaseNativePOSIX implements Linux {
     LinuxPOSIX(LibCProvider libcProvider, POSIXHandler handler) {
         super(libcProvider, handler);
 
+        statVersion = getStatVersion();
+        if (statVersion == -1) {
+            use_fxstat64 = false;
+            use_lxstat64 = false;
+            use_xstat64 = false;
```

Comment:
As @enebo mentioned, perhaps we should reduce this to a single boolean, since these functions are going to all be present or all not be present.